README.md

```diff
@@ -932,54 +933,47 @@ Google additional consent is supported in our mobile SDKs and is stored in the `
 
 ## Global Privacy Platform Multi-State Privacy (MSPS) Support for OTT
 
-Starting with version 7.3.0, if your configuration contains a ccpa campaign, it will automatically set GPP data. Unless configured otherwise, the following MSPA attributes will default to:
+Starting with version 7.6.0, if your configuration contains a ccpa campaign, it will automatically set GPP data. 
```

Comment:
the new version you don’t have anymore the option to add these:
- MspaCoveredTransaction: `NO`
- MspaOptOutOptionMode: `NOT_APPLICABLE`
- MspaServiceProviderMode: `NOT_APPLICABLE`
for this reason I removed them

Reply:
@carmelo-iriti there's a chance we'll have to add it back, because there's no GPP support for OTT and clients need a way to set up their config (those `Mspa...` flags). 

But don't add it back yet, let's see if there's a way around it (or if there are users using it at all).